runtime/onert/core/src/compiler/pass/ConstantInsertionPass.cc

```diff
@@ -40,9 +40,27 @@ void ConstantInsertionPass::callback(const ir::OperationIndex &node_index, ir::I
     auto &object = _graph.operands().at(input);
 
     const auto key = ReplaceKey{input, factor};
```

Comment:
@hseok-oh 

I cannot find my previous comment. Maybe it is not written.

The declaration of `key` is used immediately after this line before this PR.

With this change, the use of `key` is a far below.

Please move line 42 to immediately before line 64.

Reply:
Right. I did not move this to reduce code change.

Reply:
I'll update